services/vision/mlvision/ml_model.go

```diff
@@ -108,12 +108,25 @@ func registerMLModelVisionService(
 }
 
 // Unpack output based on expected type and force it into a []float64.
+// If "" is selected as the name argument, we will attempt to unpack the first entity in the map.
 func unpack(inMap map[string]interface{}, name string) ([]float64, error) {
 	var out []float64
-	me := inMap[name]
+	var me interface{}
+	if name != "" {
+		me = inMap[name]
+	} else {
+		if len(inMap) < 1 {
+			return nil, errors.New("could not unpack nonexistent first tensor")
+		}
+		for _, v := range inMap {
+			me = v
+			break
```

Comment:
Go purposefully randomizes the order that it iterates through a map every time you run it. It is not deterministic. This will not work how you want

Reply:
https://stackoverflow.com/questions/55925822/why-are-iterations-over-maps-random

Reply:
Okok makes sense but I hate it. Changed to only do this if we literally have just the one thing.